model/gen-ai/registry.yaml

```diff
@@ -177,6 +177,71 @@ groups:
               brief: 'Output tokens (completion, response, etc.)'
         brief: The type of token being counted.
         examples: ['input', 'output']
+      - id: gen_ai.agent.id   # alternatives: assistant (openai)
+        stability: experimental
+        type: string
+        brief: The unique identifier of the GenAI agent.
+        examples: ['asst_5j66UpCpwteGg4YSxUnt7lPY']
+      - id: gen_ai.agent.name
+        stability: experimental
+        type: string
+        brief: Human-readable name of the GenAI agent provided by the application.
+        examples: ["Math Tutor", "Fiction Writer"]
+      - id: gen_ai.agent.description
+        stability: experimental
+        type: string
+        brief: Free-form description of the GenAI agent provided by the application.
+        examples: ["Helps with math problems", "Generates fiction stories"]
+      - id: gen_ai.agent.tools
+        stability: experimental
+        type: string[]
+        brief: List of tool names available to the agent.
+        examples:
+          - ["Flights", "Search", "Code"]
+          - ["calc"]
+      - id: gen_ai.agent.status
+        stability: experimental
+        type:
+          members:
+            - id: completed
+              value: "completed"
+              brief: 'The agent has been completed'
+              stability: experimental
+            - id: failed
+              value: "failed"
+              brief: 'The agent was unable to complete the task due to an error.'
+              stability: experimental
+        brief: The status of the agent.
+      - id: gen_ai.agent.tool.name
```

Comment:
does it need to be in the agent namespace? why not just `gen_ai.tool.name` ? It's equally useful for apps doing tool+model without agents